COPYING.fastGL

```diff
@@ -0,0 +1,20 @@
+//*******************************************
+//   Copyright (C) 2014 by Ignace Bogaert   *
+//*******************************************
+
+// This software package is based on the paper
+//    I. Bogaert, "Iteration-Free Computation of Gauss-Legendre Quadrature Nodes and Weights",
+//    to be published in the SIAM Journal of Scientific Computing.
+
+// The main features of this software are:
+// - Speed: due to the simple formulas and the O(1) complexity computation of individual Gauss-Legendre 
+//   quadrature nodes and weights. This makes it compatible with parallel computing paradigms.
+// - Accuracy: the error on the nodes and weights is within a few ulps (see the paper for details).
+
+// Disclaimer:
+// THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESSED OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED 
+// WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE REGENTS OR 
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+// BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+// HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR 
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
also reference it our LICENSE file, into CMakeLists.txt, and finally in developer_guide/architecture.rst